spec/components/trainees/confirmation/degrees/view_preview.rb

```diff
@@ -4,30 +4,38 @@ module Confirmation
     module Degrees
       class ViewPreview < ViewComponent::Preview
         def with_one_uk_degree
-          render_component(Trainees::Confirmation::Degrees::View.new(degrees: single_uk_degree))
+          render_component(Trainees::Confirmation::Degrees::View.new(trainee: mock_trainee(degrees: single_uk_degree)))
         end
 
         def with_multiple_uk_degree
-          render_component(Trainees::Confirmation::Degrees::View.new(degrees: multiple_uk_degrees))
+          render_component(Trainees::Confirmation::Degrees::View.new(trainee: mock_trainee(degrees: multiple_uk_degrees)))
         end
 
         def with_one_non_uk_degree
-          render_component(Trainees::Confirmation::Degrees::View.new(degrees: single_non_uk_degree))
+          render_component(Trainees::Confirmation::Degrees::View.new(trainee: mock_trainee(degrees: single_non_uk_degree)))
         end
 
         def with_multiple_non_uk_degree
-          render_component(Trainees::Confirmation::Degrees::View.new(degrees: multiple_non_uk_degrees))
+          render_component(Trainees::Confirmation::Degrees::View.new(trainee: mock_trainee(degrees: multiple_non_uk_degrees)))
         end
 
         def with_a_mixture_of_uk_and_non_uk_degrees
-          render_component(Trainees::Confirmation::Degrees::View.new(degrees: mixture_of_uk_and_non_uk_degrees))
+          render_component(Trainees::Confirmation::Degrees::View.new(trainee: mock_trainee(degrees: mixture_of_uk_and_non_uk_degrees)))
         end
 
       private
 
+        def mock_trainee(degrees:)
```

Comment:
Can we use the factories here?

Reply:
We cant use factories for the previews because we would have to move the gem into the production gem group to be able to view the preview components in heroku or PAAS

Reply:
https://github.com/DFE-Digital/register-trainee-teachers/pull/85#discussion_r503527426